events/events.shacl.ttl

```diff
@@ -68,24 +82,35 @@
            [sh:class premis:SoftwareAgent] 
            [sh:class premis:HardwareAgent]
         ) ;
-        sh:class premis:Object ;
         sh:name "was attributed to"@en ;
-        sh:name "was toegekend aan"@nl;
+        sh:name "est attribué à"@fr ;
+        sh:name "is toegekend aan"@nl;
```

Comment:
een event ligt in het verleden, dus lijkt mij verleden tijd OK? En in het Engels is het nog verleden tijd?
Is "werd toegekend aan" niet correcter?

Reply:
Is goed! Ik was wat in de war met de verschillende tijden bij events.